### PR TITLE
[DO NOT MERGE] [x86/Linux] CI unittest

### PR DESCRIPTION
**Pull request by hseok-oh**
- Copy test binary from x86 Windows build
- Copy CoreFX managed from arm CoreFX managed build
- Add archival

### CODE REVIEW

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
related issue: #9903

**Comment by BruceForstall** [comment]
@jashook @RussKeldorph PTAL

File: netci.groovy [hunk 1]
```diff
@@ -2142,6 +2151,53 @@ combinedScenarios.each { scenario ->
                                         }
                                     }
                                 }
+                                else if (architecture == 'x86' && os == 'Ubuntu') {
```
Comment (RussKeldorph) [inline netci.groovy]: I believe there is enough redundancy with the code just above that we should factor out the differences and merge the blocks with appropriate `if`'s.  If you strongly disagree, let me know.

If you're worried about affecting working code, let's stage these changes in the dev/unix_test_workflow branch.  Once they work, we can port the change to master.

@jashook should be able to help with that.

**Comment by jashook** [comment]
It looks like this PR is basic its logic on the existing arm32 linux jobs which copy from existing latest coreFX and coreCLR jobs artifacts. I think the preferred approach expand upon the existing x64 and arm64 linux infrastructure. The changes then would be similar to #12311, except less profound because the x86 runs should be able to run on the x64 linux vms.

Note the logic to do this starts on on [netci.groovy:2232](https://github.com/dotnet/coreclr/blob/master/netci.groovy#L2232).

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test Tizen armel Cross Debug Build please
@dotnet-bot test Windows_NT x64 Release Priority 1 Build and Test please

**Comment by hseok-oh** [comment]
I've updated
- For x86 test job, I expanded existing x64 and arm64 linux infrastructure

@RussKeldorph @jashook PTAL

**Comment by karelz** [comment]
What is status of this PR? Are we blocked on another round of code review from @RussKeldorph @jashook?

**Comment by jashook** [comment]
@hseok-oh do you mind opening a PR to Pr this into the [dev/unix_test_workflow branch](https://github.com/dotnet/coreclr/tree/dev/unix_test_workflow). It is large enough to warrant testing in that branch before merging.

**Comment by hseok-oh** [comment]
@jashook OK. I'll make new PR

**Comment by karelz** [comment]
Do we still need this PR in such case?

**Comment by jashook** [comment]
For now it is worth keeping this pr open.

**Comment by hseok-oh** [comment]
@jashook I've opend new PR into dev/unix_test_workflow branch.

**Comment by jashook** [comment]
Will close this pr until the changes are tested in #13681

**Pull request closed without merging**